src/OrbitGl/App.h

```diff
@@ -84,15 +85,17 @@ class OrbitApp final : public DataViewFactory,
                        public orbit_capture_client::CaptureListener,
                        public orbit_data_views::AppInterface {
  public:
-  explicit OrbitApp(orbit_gl::MainWindowInterface* main_window,
-                    MainThreadExecutor* main_thread_executor,
-                    const orbit_base::CrashHandler* crash_handler,
-                    orbit_metrics_uploader::MetricsUploader* metrics_uploader = nullptr);
+  explicit OrbitApp(
+      orbit_gl::MainWindowInterface* main_window, MainThreadExecutor* main_thread_executor,
+      const orbit_base::CrashHandler* crash_handler,
+      const orbit_statistics::BinomialConfidenceIntervalEstimator* confidence_interval_estimator,
```

Comment:
See a more lengthier version of this comment below, but TLDR I dont think we should pass the estimator here, but rather simply instantiate it here in App directly.